### DISCUSSION TITLE
AdGuard Home rDNS (PTR) Settings

### DISCUSSION THREAD

**Discussion by whyisthisbroken** [AdguardTeam/AdGuardHome#4476] [Q&A]
So, since the last update, i dont understand anymore the correct settings for rDNS.

Could someone explain where i put the rDNS Server and what should i set in the Upstream Field ?

The Wiki is not really helpful or i´m simply stupid...

Before the 1.108 Update - everything works fine, and now all IPv6 dont resolve anymore...

I have a Fritzbox, so here are the settings I think are correct:
Here I don't get an error, but IPv6 is not resolved...

```
Upstream Server:
Nothing ?!?

Private reverse DNS servers:
192.168.178.1
fd00::464e:xxxx:xxxx:xxxx
```

The Following settings cause an Error:

```
Private reverse DNS servers:
[/168.192.in-addr.arpa/]192.168.178.1
[/d.f.ip6.arpa/]192.168.178.1
```

If i apply the settings this error appear.
**Error: control/dns_config | validating private upstream servers: no default upstreams specified | 400**

![Screenshot 2022-04-08 175734](https://user-images.githubusercontent.com/96284773/162479062-2cdb9345-b6d3-475c-942a-2d275b93fd58.jpg)

What does it mean ? 

It would be nice if someone could help me, it drives me crazy....

**Comment by ainar-g** [comment]
Hello.  Looks like you should just leave `192.168.178.1` in the field.  The error tells you that you haven't put a default upstream, only entering upstreams for two domain names.  So if a PTR request for one of the ranges besides those comes, AGH won't know where to forward it.

**Reply by EugeneOne1** [reply]
We've got an issue (#4292) about it actually.

**Comment by whyisthisbroken** [comment]
Here is my current setup:

Upstream Server:
![Screenshot 2022-04-08 180541](https://user-images.githubusercontent.com/96284773/162480684-265bcf1e-4d25-4fee-ae10-84de9499d141.jpg)
Bootstrap Server:
![Screenshot 2022-04-08 180554](https://user-images.githubusercontent.com/96284773/162480696-68dfe28e-9dbd-4a64-b05a-8855bed2e2ed.jpg)
With this PTR the ipv4 works - but not ipv6
![Screenshot 2022-04-08 180612](https://user-images.githubusercontent.com/96284773/162480714-71416967-5901-410f-9f89-cd6f17cbbfb1.jpg)
With these settings i get the error:
![Screenshot 2022-04-08 180631](https://user-images.githubusercontent.com/96284773/162480781-42923dcb-6e45-4f6a-888e-f808edc035cd.jpg)

So, if there is an issue - could you please tell me if these settings are correct and they should work without the issue ?

**Reply by ainar-g** [reply]
Please elaborate on what you mean when you say that “ipv4 works - but not ipv6”.  What are the queries that work or don't work?

**Reply by whyisthisbroken** [reply]
I mean - the ipv4 is resolved... with the rDNS name that comes from the fritzbox. But not the ipv6. they are listed with the fd00... IP in the client list...

Sorry for my broken english - I hope it is understandable

For Example here a Screenshot:
![Screenshot 2022-04-08 183125](https://user-images.githubusercontent.com/96284773/162483974-f292ef6b-c80c-4493-be8a-e413bebd1efd.jpg)

Or is this maybe a hardware issue ? that some network devices doesnt support the rDNS ?
Here is my Notebook - the other one is my Pixel 5... 
![Screenshot 2022-04-08 183624](https://user-images.githubusercontent.com/96284773/162484701-b335d050-b4b9-4fd9-8b1b-36fd810cf269.jpg)

NSLOOKUP from this 2 devices:
![Screenshot 2022-04-08 183731](https://user-images.githubusercontent.com/96284773/162484788-a022a5eb-d5e5-4342-ab62-be8f095307fe.jpg)

**Reply by ainar-g** [reply]
@whyisthisbroken, well, if some IPv6 addresses are being resolved while others aren't, that feels like an upstream (FritzBox) or a client (Pixel) issue.  At least to me.  Perhaps, your Pixel doesn't send a correct hostname to the router when it joins the network and sends its DHCP requests, while the notebook does.

**Reply by mercyground** [reply]
your google pixel uses SLAAC not DHCPv6. Google refuses to support DHCPv6. see here : https://issuetracker.google.com/issues/36949085

its a long standing issue and its up to google to fix this.

**Reply by whyisthisbroken** [reply]
Well ,this maybe is a Pixel Problem - but the other FD00 ipv6 Adresses will also not resolve in my Network. This is what i dont understand.
